lib/swaggerValidator/ajv.ts

```diff
@@ -157,7 +157,16 @@ export const ajvEnableAll = (ajv: Ajv, jsonLoader: JsonLoader) => {
   ajvEnableInt32AndInt64Format(ajv);
   ajvEnableDurationFormat(ajv);
   ajvEnableDateTimeRfc1123Format(ajv);
-  ajvAddFormatsDefaultValidation(ajv, "string", ["byte", "password", "file", "base64url"]);
+  ajvAddFormatsDefaultValidation(ajv, "string", [
+    "byte",
+    "password",
+    "file",
+    "base64url",
+    "",
```

Comment:
what does the empty string do?

Reply:
For the format added here, we just ignore validation coz it doesn't have knowledge to validate.